stories/index.js

```diff
@@ -3,8 +3,8 @@ import { storiesOf, action, linkTo } from '@kadira/storybook';
 import { host } from 'storybook-host';
 
 import SnoozePopup from '../src/lib/components/SnoozePopup';
-import MainPanel from '../src/lib/components/MainPanel';
-import ManagePanel from '../src/lib/components/ManagePanel';
+// import MainPanel from '../src/lib/components/MainPanel';
+// import ManagePanel from '../src/lib/components/ManagePanel';
```

Comment:
Deleted and force pushed. We can always dig through Git history if we need to recreate those two import statements. xoxo